src/dex-ui/pages/AddLiquidity/AddLiquidity.tsx

```diff
@@ -265,6 +295,13 @@ const AddLiquidity = (): JSX.Element => {
       : "-";
   }, [poolState]);
 
+  const poolRatio = useCallback(() => {
```

Comment:
In the future, we should refactor this to calculate the `poolRatio` on every render without the need for the `useCallback`.

Reply:
Noted